lib/go-tc/crstates.go

```diff
@@ -25,8 +25,8 @@ import (
 
 // CRStates includes availability data for caches and delivery services, as gathered and aggregated by this Traffic Monitor. It is designed to be served at an API endpoint primarily for Traffic Routers (Content Router) to consume.
 type CRStates struct {
-	Caches          map[CacheName]IsAvailable                       `json:"caches"`
-	DeliveryService map[DeliveryServiceName]CRStatesDeliveryService `json:"deliveryServices"`
+	Caches          map[string]IsAvailable             `json:"caches"`
+	DeliveryService map[string]CRStatesDeliveryService `json:"deliveryServices"`
```

Comment:
So, I haven't done a full review, just a once-over of the code mostly looking at how it interfaces with ATS, but why did this change? The CacheName and DeliveryServiceName types generally improve readability and safety, since you'd have to cast to and from them in order to convert them. They help put arguments in context and improve the information in compile errors. I'm curious what the reasoning here.

Reply:
/facepalm. I'll leave this up, but it looks like I commented at the precise moment that Rob did. He said basically the same thing, so feel free to respond only to his comments.

Reply:
Yeah, that was exactly my plan